covid-19.md

```diff
@@ -6,29 +6,34 @@ title: Extra Support for COVID-19 Research
 OSG Support for COVID-19 Research
 =================================
 
-Like many others in the research computing community, OSG is helping to organize
-extra resources to be made available to COVID-19 research!  These resources include:
+Please contact us at <mailto:help@opensciencegrid.org> if your research group could benefit
+from computing resources or consulting related to COVID-19, or if you would like to support COVID-19
+research by sharing your own capacity.
+
+OSG is helping to organize extra resources to be made available to COVID-19 research!  These resources include:
 
-- **Consultation and facilitation** for quickly scaling up and automating workflows.
-- **Expertise** on using high throughput computing for maximizing research throughput.
-- **Access to computing capacity** through the [OSG Connect](https://support.opensciencegrid.org/support/home)
-  service which provides a batch system submit environment based on HTCondor.  The OSG aggregates
+- **Proactive consultation and facilitation** for quickly scaling up and automating workloads with up to [thousands of 
+concurrently-running jobs](https://gracc.opensciencegrid.org/dashboard/db/covid-19-research?orgId=1).
+- **Expertise** on using high throughput computing for maximizing research throughput, perhaps including 
```

Comment:
Not a big fan of "perhaps including development effort" -- the "perhaps" sounds wishy-washy.

Reply:
Hi Matt,
That's why I moved the information/link to GRACC under this heading. The information under this heading is now ALL related to sites, and information for running COVID-19 work is under the first section.

Reply:
At the very least, if you merge as-is, I noticed another grammatical error that will need to be fixed, too.

Reply:
OK, should I wait for you to commit a fix?

Reply:
no. please merge!